Raptacon/Robot-2024
# Dashboard version

## Types of changes


## Associated Issue(s)
Issue: <#110 >

## Steps to Test
1. Deploy branch to robot
2. Check smart dashboard on driver station


## Changelog

Added smart dashboard print statements. Config robot and swerverobot

## What?

Read from deploy.json and wrote to smart dashboard

## Why?

To ease debugging and better understand robot
